src/gsy_e/models/strategy/energy_parameters/heat_pump.py

```diff
@@ -141,20 +209,9 @@ def _calc_temp_increase_K(self, time_slot: DateTime, energy_kWh: float) -> float
 
     def _populate_state(self, time_slot: DateTime):
         # order matters here
```

Comment:
I think that you can remove this from here, as the two remaining calls do not depend on each other.

Reply:
Remnant from debugging, removing

Reply:
This is still there.

Reply:
Missed this, removing